docs/MO_DG/IR_and_opsets.md

```diff
@@ -1,11 +1,9 @@
-# Deep Learning Network Intermediate Representation and Operation Sets in OpenVINO™ {#openvino_docs_MO_DG_IR_and_opsets}
+# Deep Learning Network Intermediate Representation and Operation Sets in OpenVINO {#openvino_docs_MO_DG_IR_and_opsets}
 
-This document provides essential information on the format used for representation of deep learning models in OpenVINO™ toolkit and supported operation sets.  
+This article provides essential information on the format used for representation of deep learning models in OpenVINO toolkit and supported operation sets.  
 
 ## Overview of Artificial Neural Networks Representation
 
```

Comment:
The summary sentence is still helpful. I suggest we add it back.

Reply:
Definitely. , "Paragraph", IMHO, doesn't fit the documentation.  It connotes a large block of unseparated text.